lib/shoryuken/middleware/server/exponential_backoff_retry.rb

```diff
@@ -12,14 +12,19 @@ def call(worker, queue, sqs_msg, body)
 
           started_at = Time.now
           yield
-        rescue
+        rescue => ex
+          logger.info { "Message #{sqs_msg.message_id} will attempt retry due to error: #{ex.message}" }
```

Comment:
good point.  I'll make that change. I had that thought with the debug statement below, but not the informational one.  No need to see the same error twice in the logs. 
I think debug already gets potentially noisy as it is.  I think the error is very useful.  Maybe `warn`?